doc/user-guide/pandas.rst

```diff
@@ -110,6 +110,29 @@ work even if not the hierarchical index is not a full tensor product:
     s[::2]
     s[::2].to_xarray()
 
+Lossless and reversible conversion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The previous ``Dataset`` example shows that the conversion is not reversible (lossy roundtrip) and
+that the size of the ``Dataset`` increases.
+
+Particularly after a roundtrip, the following deviations are noted:
+
+- a non-dimension Dataset ``coordinate`` is converted into ``variable``
+- a non-dimension DataArray ``coordinate`` is not converted
+- ``dtype`` is not allways the same (e.g. "str" is converted to "object")
+- ``attrs`` metadata is not converted
+
+To avoid these problems, the third-party `ntv-pandas`__ library offers lossless and reversible conversions between
+``Dataset``/ ``DataArray`` and pandas ``DataFrame`` objects.
+
+This solution is particularly interesting for converting any ``DataFrame`` into a ``Dataset`` (the converter find the multidimensional structure hidden by the tabular structure).
+
+The `ntv-pandas examples`__ show how to improve the conversion for the previous ``Dataset`` example and for more complex examples.
+
+__ https://github.com/loco-philippe/ntv-pandas
+__ https://github.com/loco-philippe/ntv-pandas/tree/main/example
+
```

Comment:
```suggestion
```

It's maybe better to inline these links, especially now that there is more than one.